data/demographic/census/index.html

```diff
@@ -9,20 +9,18 @@
 categories: []
 2020Census:
   downloads:
-    - url: "https://drive.google.com/uc?id=1_Gp0JVHFbjw5OqDcFGbFjo9b-lJbzRsW&export=download"
-      label: 'File Geodatabase: Census Data package for Utah'
-    - url: "https://drive.google.com/uc?id=1Efr1XXq5w4yTDa-KwLFL8oFrn-2Z8Afq&export=download"
-      label: 'Shapefile: Census Blocks for Utah'
-    - url: "https://drive.google.com/uc?id=1pSy_UwEybUFf3DFUpi1Zqhsj3LMN7gAn&export=download"
-      label: 'Shapefile: Census Block Groups for Utah'
-    - url: "https://drive.google.com/uc?id=1PBJg5DG95aGHGDbTff_wzWnM07SYjvRP&export=download"
-      label: 'Shapefile: Census Tracts for Utah'
-    - url: "https://drive.google.com/uc?id=1xgk1Jmnbl-kMSpkuoeUktX6dFgGJ1W4V&export=download"
-      label: 'Shapefile: Census Places for Utah'
-    - url: "https://drive.google.com/uc?id=1thr46UZOe2kZo97Pma2GOSMHQlgcYSrh&export=download"
-      label: 'Shapefile: Census Counties for Utah'
-    - url: "https://drive.google.com/uc?id=1WldVUwSCRgB8frB4clSf18RSG8p5uqas&export=download"
-      label: 'Shapefile: Census State Boundary for Utah'
+    - url: https://opendata.gis.utah.gov/datasets/utah-census-blocks-2020
+      label: 'Utah Census Blocks 2020'
+    - url: https://opendata.gis.utah.gov/datasets/utah-census-block-groups-2020
+      label: 'Utah Census Block Groups 2020'
+    - url: https://opendata.gis.utah.gov/datasets/utah-census-tracts-2020
+      label: 'Utah Census Tracts 2020'
+    - url: https://opendata.gis.utah.gov/datasets/utah-census-places-2020
+      label: 'Utah Census Places 2020'
+    - url: https://opendata.gis.utah.gov/datasets/utah-census-counties-2020
+      label: 'Utah Census Counties 2020'
+    - url: https://opendata.gis.utah.gov/datasets/utah-census-state-2020
+      label: 'Utah Census State 2020'
```

Comment:
**issue**: the current download options is a 1:1 where this needs 1:M. That would be a nice enhancement to the include and rick has been asking for it also. For now I suppose this is ok but it will be a little confusing to have to then understand the hub ui to get to the download.